ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { I18nContext } from '../../../../../contexts/i18n';
+import {
+  Box,
+  ButtonIcon,
+  ButtonIconSize,
+  IconName,
+} from '../../../../../components/component-library';
+import type { StyleUtilityProps } from '../../../../../components/component-library/box';
+
+import {
+  BackgroundColor,
+  BlockSize,
+  Display,
+  FlexDirection,
+  IconColor,
+  BorderRadius,
+} from '../../../../../helpers/constants/design-system';
+import { useScrollRequired } from '../../../../../hooks/useScrollRequired';
+import { updateConfirm } from '../../../../../ducks/confirm/confirm';
+
+interface ContentProps extends StyleUtilityProps {
+  /**
+   * Elements that go in the page content section
+   */
+  children: React.ReactNode | React.ReactNode[];
+}
```

Comment:
Not sure, but does this needs to extend `StyleUtilityProps` ?

Reply:
oh right. Good catch. We removed need for ...props here so the extension is no longer needed. Updated 3c4d58949cbbbca1676b5e774ab442a43e21c509